### PR TITLE
oas task links

### PR DESCRIPTION
**Pull request by nabil386**
### Description
add links to oas tasks 
List of proposed changes:

- 
-

### What to test for/How to test
dashboard test has been updated

### CODE REVIEW

**Review by amirpourpanahi — APPROVED** [review]

File: contents/DashboardBenefitCardConstants.js [hunk 1]
```diff
@@ -79,6 +82,9 @@ const ACTIVE_OAS = {
       description: 'July 1, 2021',
     },
   ],
+  editAddressLink: t.oasBenefitEditAddressLink,
```
Comment (arsanig) [inline contents/DashboardBenefitCardConstants.js:85]: I don't think these will be passed in as part of the APIs in order for them to be a part of the contents

Comment (nabil386) [reply contents/DashboardBenefitCardConstants.js:85]: they won't be but if edit address link is different for each benefit then we need to pass it as prop

**Review by arsanig — CHANGES_REQUESTED** [review]

File: locales/fr.js [hunk 2]
```diff
@@ -289,10 +289,26 @@ export default {
...
+    'https://srv136.services.gc.ca/sc/msca-mdsc/portal-portail/pro/socv-vecc/socv/earnings-contributions',
+  taxDeductionsTaskLink:
+    'https://srv136.services.gc.ca/sc/msca-mdsc/portal-portail/pro/vupi-vupi/view-tax-deduction.action',
+  delayOasPensionTaskLink:
+    'https://srv136.services.gc.ca/sc/msca-mdsc/portal-portail/pro/vupi-vupi/delay-oas-pension.action',
+  giveConsentTaskLink:
+    'https://srv136.services.gc.ca/sc/msca-mdsc/portal-portail/pro/vupi-vupi/view-consent-to-communicate.action',
+  taxSlipTaskLink:
     'https://srv136.services.gc.ca/sc/msca-mdsc/portal-portail/pro/tiso-frfd/tax-slip',
-  taxSlipMailingLink:
```
[the remark this answers is not in the document]
Comment (nabil386) [reply locales/fr.js:294]: it is not removed it exists at line 302

**Review by arsanig — APPROVED** [review]
Awesome! Please squash when merging

**Pull request merged**